src/lib/components/SparklineChart/_styles.scss

```diff
@@ -61,10 +70,21 @@ web-sparkline-chart {
     }
   }
 
+  svg {
+    pointer-events: none;  // otherwise mouseout generates events for every part of SVG
+    display: block;
+  }
+
+  .sl-cursor {
+    stroke-linecap: round;
+  }
+
   .path {
-    stroke-dasharray: 1000;
-    stroke-dashoffset: 1000;
-    animation: sparkline-dash-animation 300ms linear forwards;
+    stroke-linecap: round;
+    // TODO(robdodson): This doesn't animate even in the old site.
```

Comment:
I'm cool just removing this stuff. I think Eric added it originally and my knowledge of SVG animation is nil :P

Reply:
Removed